gulpfile.js

```diff
@@ -210,9 +213,9 @@ gulp.task("watch", ["serve"], function () {
     );
     gulp.watch(
         [
-            path.join(ROOT_DIR, "**", "*.yml"),
-            path.join(SOURCE_DIR, "**", "*.html"),
-            path.join(SOURCE_DIR, "**", "*.md"),
+            path.join(DATA_DIR, "**", "*.yml"),
```

Comment:
We want to regenerate the site if the configs change too, and `DATA_DIR` doesn't include `CONFIG_DIR`.